frontend/public/index.html

```diff
@@ -9,6 +9,9 @@
       name="description"
       content="Web site created using create-react-app"
     />
+
+    <script src="https://kit.fontawesome.com/71bead3b94.js" crossorigin="anonymous"></script>
```

Comment:
What does this do?

Reply:
This allows me to pull various icons and texts from a font website. https://fontawesome.com/

Reply:
Ah ok cool!